core/server/worker/src/main/java/alluxio/worker/netty/BlockReadHandler.java

```diff
@@ -198,7 +198,7 @@ private void openBlock(BlockReadRequestContext context, Channel channel) throws
         // Sends an empty buffer to the client to make sure that the client does not timeout when
         // the server is waiting for the UFS block access.
         channel.writeAndFlush(new RPCProtoMessage(heartbeat));
-      } while (retryPolicy.attemptRetry());
+      } while (retryPolicy.attempt());
```

Comment:
We shouldn't use `do .. while` anymore since that will cause an extra retry

Reply:
Done